crates/sgx_validation/src/config.rs

```diff
@@ -0,0 +1,124 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize};
+use sgx::parameters::Features;
+
+#[derive(Clone, Deserialize, Debug, Default, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Config {
+    /// Values for `mrsigner` in the report body.
+    /// This is the list of public keys which have signed the Enarx binary.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_hex")]
+    pub enarx_signer: Option<Vec<Vec<u8>>>,
+
+    /// Values for `features`.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_features")]
+    pub features: Option<u64>,
+
+    /// Value allowed for `cpusvn`.
+    pub cpu_svn: Option<Vec<u8>>,
```

Comment:
```suggestion
    /// Minimum value allowed for `cpusvn`.
    pub cpu_svn: Option<Vec<u8>>,
```

Reply:
Fixed

Reply:
Ok, we can't check against the cpu_svn that easily. See https://github.com/profianinc/steward/issues/112 and the linked pdf document.